vividus-extension-selenium/src/main/java/org/vividus/proxy/har/HarManager.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.proxy.har;
+
+import com.browserup.harreader.model.Har;
+import com.browserup.harreader.model.HarLog;
+
+import org.vividus.proxy.IProxy;
+import org.vividus.testcontext.TestContext;
+
+public class HarManager
+{
+    private final TestContext testContext;
+    private IProxy proxy;
+
+    public HarManager(IProxy proxy, TestContext testContext)
+    {
+        this.testContext = testContext;
+        this.proxy = proxy;
+    }
+
+    public Har takeHar()
+    {
+        return createHarWithUniqueData(proxy.getRecordedData());
+    }
+
+    private Har createHarWithUniqueData(Har har)
+    {
+        Har newHar = addHarLog(har);
+        Har savedHar = testContext.get(Har.class);
+        if (null != savedHar)
+        {
+            return removeTheSameValues(newHar, savedHar.getLog());
```

Comment:
what will happen
1) when the first failure happens in scenario 1, and the next one - in scenario 2?
2) shouldn't be the current har saved to test context?